ignite/metrics/precision.py

```diff
@@ -41,32 +41,33 @@ def __init__(
     def reset(self) -> None:
         if self._is_multilabel:
             init_value = 0.0 if self._average else []
-            kws = {"dtype": torch.float64, "device": self._device}
-            self._true_positives = torch.tensor(init_value, **kws)
-            self._positives = torch.tensor(init_value, **kws)
+            self._true_positives = torch.tensor(init_value, dtype=torch.float64, device=self._device)
+            self._positives = torch.tensor(init_value, dtype=torch.float64, device=self._device)
         else:
             self._true_positives = 0
             self._positives = 0
 
         super(_BasePrecisionRecall, self).reset()
 
     def compute(self) -> Union[torch.Tensor, float]:
-        is_scalar = not isinstance(self._positives, torch.Tensor) or self._positives.ndim == 0
+        is_scalar = (
+            not isinstance(self._positives, torch.Tensor) or self._positives.ndim == 0  # type: ignore[attr-defined]
+        )
         if is_scalar and self._positives == 0:
             raise NotComputableError(
                 "{} must have at least one example before it can be computed.".format(self.__class__.__name__)
             )
 
         if not (self._type == "multilabel" and not self._average):
-            if not self._is_reduced:
-                self._true_positives = idist.all_reduce(self._true_positives)
-                self._positives = idist.all_reduce(self._positives)
+            if not self._is_reduced:  # type: ignore[has-type] # mypy cann't determine type
```

Comment:
I had that this error from time to time. Let's remove it too and see if we can not do otherwise.